Source/JavaScriptCore/wasm/WasmIRGeneratorHelpers.h

```diff
@@ -163,6 +163,37 @@ static inline void emitThrowImpl(CCallHelpers& jit, unsigned exceptionIndex)
     });
 }
 
+static inline void emitStructNewImpl(CCallHelpers& jit, unsigned typeIndex)
```

Comment:
We should inline this allocation, either now or in a followup. If a followup, let's open a bug for it and link to it here.